### DISCUSSION TITLE
Download filenames

### DISCUSSION THREAD

**Discussion by disl-pdavid** [ckan/ckan#6866] [Q&A] [answered]
[edit] - fixed in #6888!

Hello again all!

I've got a stock CKAN instance from master branch `ckan 2.10.0a0`.

With just the stock install and no other extensions I can create a new dataset and upload resources.
But, downloading a resource gives the filename as a UUID string instead of the uploaded filename:

![image](https://user-images.githubusercontent.com/105015605/168899364-2eb51ee3-03dc-43d7-bcac-d07e064900de.png)

Clicking either the link or download button prompts Firefox (`100.0.1 (64-bit)`)
:
![image](https://user-images.githubusercontent.com/105015605/168899538-53019b7b-de06-442f-928c-9a341bb136c5.png)

When I would have expected the filename instead `628_2022_hyd_sum.csv`.

Is this a simple setting I'm missing somehwere?

**Comment by disl-pdavid** [comment]
Just tried this with `tags/ckan-2.9.5` and it _does_ work as expected (stock docker install).

**Comment by smotornyuk** [comment, marked as answer]
Thank you for reporting the issue. It will be fixed by ~~#6866~~ #6888

**Reply by disl-pdavid** [reply]
Sorry, the link you provided links back to this discussion - did you mean to include a commit or issue?

**Reply by smotornyuk** [reply]
Yep, sorry. I meant #6888
